### PR TITLE
data: Ertan222

### PR DESCRIPTION
**Pull request by Ertan222**
## Changes proposed

I created my linkfree profile

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3775"></a>

### CODE REVIEW

File: data/Ertan222.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+  "name": "Ertan Taner",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Jr. Backend Developer | Animal Lover | Open Source Is Everything | Linux Enthusiast | Terminal > GUI ",
+  "avatar": "https://avatars.githubusercontent.com/u/62712555?v=4",
```
Comment (Aadarsh805) [inline data/Ertan222.json:6]: change the avatar link to `https://github.com/Ertan222.png`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/Ertan222.json (hunk 1 above)
Comment (Ertan222) [reply data/Ertan222.json:6]: Thanks for feedback, going to change it right now

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to merge, Thanks for your contribution! 🎉
<p align="center"></p>

**Pull request merged**